ossapi/ossapiv2.py

```diff
@@ -603,6 +618,18 @@ def changelog_lookup(self,
         return self._get(Build, f"/changelog/{changelog}", params)
 
 
+    # /chat
+    # ---------
+
+    @request
+    def create_new_pm(self, target_id: int, message: str, is_action: bool = False) -> CreateNewPMResponse:
```

Comment:
let's rename `target_id` to `user_id`, and `create_new_pm` to `create_pm`

Reply:
I think `target_id` is for if/when this endpoint supports group DMs and/or chat channels. `user_id` is more specific to the PM use case, so I'll update